session/session.go

```diff
@@ -1163,7 +1163,7 @@ func (s *session) ExecuteStmt(ctx context.Context, stmtNode ast.StmtNode) (sqlex
 
 	// Execute the physical plan.
 	logStmt(stmtNode, s.sessionVars)
-	recordSet, err := runStmt(ctx, s, stmt)
+	recordSet, err := runStmtWrap(ctx, s, stmt)
```

Comment:
So the binary protocol (execute statement) will still use `runStmt`? Do you need to fix it?

Reply:
Yes, it will be handled in the following works.